package.json

```diff
@@ -84,13 +84,13 @@
     "typechain": "^1.0.3",
     "typechain-target-truffle": "^1.0.2",
     "typescript": "^3.7.4",
-    "web3": "^1.2.7"
+    "web3": "^1.2.0"
   },
   "peerDependencies": {
-    "@nomiclabs/buidler": "^1.3.2",
-    "@nomiclabs/buidler-etherscan": "^1.3.2",
-    "@nomiclabs/buidler-truffle5": "^1.3.2",
-    "@nomiclabs/buidler-web3": "^1.3.2",
-    "web3": "^1.2.7"
+    "@nomiclabs/buidler": "^1.3.0",
+    "@nomiclabs/buidler-etherscan": "^1.3.0",
+    "@nomiclabs/buidler-truffle5": "^1.3.0",
+    "@nomiclabs/buidler-web3": "^1.3.0",
+    "web3": "^1.2.0"
```

Comment:
Note that web3@1.2.8 introduced a significant change to the ABI encoded (ethers@5). Not sure if it has any impact to us, or how deeply we use web3 in our plugin, but something worth checking.